docs/ops/matrix/MatMul_1.md

```diff
@@ -12,29 +12,43 @@
 
 Before matrix multiplication, there is an implicit shape alignment for input arguments. It consists of the following steps:
 
-1. If rank of an input less than 2 it is unsqueezed to 2D tensor by adding axes with size 1 to the left of the shape. For example, if input has shape `[S]` it will be reshaped to `[1, S]`. It is applied for each input independently.
+1. Applying transpositions specified by optional `transpose_a` and `transpose_b` attributes for 2D tensors and bigger.
 
-2. Applied transpositions specified by optional `transpose_a` and `transpose_b` attributes.
+2. One-dimensional tensors unsqueezing is applied for each input independently. The axes inserted in this step are not included in the output shape.
+    * If rank of the **first** input is less than 2, it is unsqueezed to 2D tensor by adding axes with size 1 at ROW_INDEX_DIM, to the **left** of the shape. For example `[S]` will be reshaped to `[1, S]`.
```

Comment:
Please avoid using "rank of input less than 2" and replace it by "1D input" or by "rank of input is 1" as we don't allow 0D tensors to be used as inputs for MatMul.

Please add interpretations for such dimension handling in steps 1 and 2 by saying that first 1D input is interpreted as a row-vector regardless of transpose_a which is ignored in this case.

The same is applicable for the second input. The second 1D input is always interpreted a column-vector.

Reply:
Done